arch/none/host/src/arch_main.c

```diff
@@ -35,3 +35,8 @@ int main(void)
     if (status != FWK_SUCCESS)
         panic();
 }
+
+int scmi_arch_deinit(void)
+{
+    return fwk_arch_deinit();
+}
```

Comment:
This is dead code we can remove it.
Also, we usually use `scmi_` for protocol related functions. It is better not to use it here.

Reply:
this is used after reworking host main loop. will move it in the right commit